app.js

```diff
@@ -1,11 +1,66 @@
 /* Imports */
 
 /* Get DOM Elements */
+const italianDropdown = document.getElementById('italian-dd');
+const mexicanDropdown = document.getElementById('mexican-dd');
+const chineseDropdown = document.getElementById('chinese-dd');
+const italianImage = document.getElementById('italian-img');
+const mexicanImage = document.getElementById('mexican-img');
+const chineseImage = document.getElementById('chinese-img');
+
+const italianCountEl = document.getElementById('italian-count');
+const mexicanCountEl = document.getElementById('mexican-count');
+const chineseCountEl = document.getElementById('chinese-count');
+
+const inputEl = document.getElementById('text');
+const buttonEl = document.getElementById('my-button');
 
 /* State */
+let italianCount = 0;
+let mexicanCount = 0;
+let chineseCount = 0;
+let text = [];
+// needs HTML spans.
 
 /* Events */
+italianDropdown.addEventListener('change', () => {
+    italianImage.src = `assets/Italian-${italianDropdown.value}.png`;
+    italianCount++;
+
+    displayStats();
+});
+
+mexicanDropdown.addEventListener('change', () => {
+    mexicanImage.src = `assets/Mexican-${mexicanDropdown.value}.png`;
+    mexicanCount++;
+
+    displayStats();
+});
+
+chineseDropdown.addEventListener('change', () => {
+    chineseImage.src = `assets/Chinese-${chineseDropdown.value}.png`;
+    chineseCount++;
+
+    displayStats();
+});
+
+buttonEl.addEventListener('click', () => {
```

Comment:
This listener still needs to clear out the input's value and called the new display() function you'll make based off the comment below.